### DISCUSSION TITLE
Memory issue encountered training DeepAR model

### DISCUSSION THREAD

**Discussion by SebastienZh** [autogluon/autogluon#2874] [Q&A]
I try to train a autogluon.timeseries.DeepARModel on my RTX3060. Unfortunately, the following warning reminded me that my RTX3060 does not have sufficient memory for training.

> CUDA out of memory. Tried to allocate 13.07 GiB (GPU 0; 6.00 GiB total capacity; 1.86 GiB already allocated; 2.21 GiB free; 2.04 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

However, the warning suggests that the parameter _max_split_size_mb_ can be tuned to allocate less memory for training.  But I don't see this parameter available in autogluon DeepARModel. Is there an alternative way to solve this memory issue?

**Comment by shchur** [comment]
This is strange, for me training DeepAR with default hyperparameters only takes 883 MB of GPU RAM. One possible reason is a large number of covariates / static features included in your dataset. Can you try excluding them from the data (only keep the target column) and seeing if the issue persists?
